app/app.js

```diff
@@ -1,18 +1,16 @@
-import Ember from 'ember';
-import Resolver from './resolver';
+import Ember, { Application } from 'ember';
 import loadInitializers from 'ember-load-initializers';
-import config from './config/environment';
-
-let App;
+import { modulePrefix, podModulePrefix } from 'ember-get-config';
```

Comment:
I like the consistency of it and it's very common to require the config. If you have to use `../` within nested directories, that becomes painful.